src/recensio/plone/profiles/default/types/Presentation_Article_Review.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        meta_type="Dexterity FTI"
+        name="TODO"
```

Comment:
Do you want to merge this thing?

Reply:
no, the TODO placeholders are to quickly find the places where some real thing has to be inserted. these are only stubs,.